src/ecalc/libraries/libecalc/common/libecalc/dto/result/results.py

```diff
@@ -107,6 +108,8 @@ class CompressorResult(EquipmentResultBase):
     recirculation_loss: TimeSeriesRate
     rate_exceeds_maximum: TimeSeriesBoolean
     outlet_pressure_before_choking: TimeSeriesFloat
+    requested_inlet_pressure: Dict[datetime, TimeSeriesFloat]
```

Comment:
do we want the `datetime` here?

Reply:
I think we need it if we are collecting results for temporal models?

Reply:
but, since we "assume" that all temporal models have this value (even though it might be NaN/null), it is a "compressor property", and we do not care about the implementation here, we just want to show that value across time. That is in general the idea for these common properties..

Reply:
datetime is removed, only returning TimeSeriesFloat, i.e. pressure vs. time.